test_store.py

```diff
@@ -23,8 +29,7 @@ def test_add_todo(db):
 def test_get_single_todo(db):
     db.add_todo("gym")
     db.add_todo("sleep")
-    tasks = db.get_todos(task_id=2)
-    task = tasks[0]
+    task = db.get_todos(task_id=2)
```

Comment:
this was a wrong test, as mentioned above a get_todos with a single id should return a task object, not a list